windows/container.go

```diff
@@ -90,16 +83,16 @@ func (c *container) Start(ctx context.Context) error {
 	}
 
 	c.setStatus(runtime.RunningStatus)
-	c.sendEvent(c.ctr.ID(), events.RuntimeEvent_START, c.ctr.Pid(), 0, time.Time{})
+	//	c.sendEvent(c.ctr.ID(), events.RuntimeEvent_START, c.ctr.Pid(), 0, time.Time{})
```

Comment:
`r.events <- &events.TaskStart{...}`

Would need to add a `go r.forward(client)` similar to the one in the linux shim though.

Similar ideas for the other instances.